run.py

```diff
@@ -82,6 +82,26 @@
     get_save_dir,
 )
 
+import warnings
+
+
+def id_checker(style_id: Optional[int], speaker_id: Optional[int]) -> int:
+    """
+    style_idとspeaker_id両方ともNoneかNoneでないかをチェックし、
+    どちらか片方しかNoneが存在しなければstyle_idを返す
+    """
+    if (style_id is None and speaker_id is None) or (
+        style_id is not None and speaker_id is not None
+    ):
+        raise HTTPException(
+            status_code=400, detail="speakerとstyle_idが両方とも存在しないか、両方とも存在しています。"
+        )
+    if speaker_id is not None:
+        warnings.warn("speakerは非推奨です。style_idの利用を推奨しています。")
+        style_id = speaker_id
+        speaker_id = None
+        return style_id
```

Comment:
あれこれだと正しく動かないはず･･･？
こうじゃないですかね

```suggestion
    if speaker_id is not None:
        warnings.warn("speakerは非推奨です。style_idの利用を推奨しています。")
        style_id = speaker_id
        speaker_id = None
    return style_id
```